x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/stateful_event.tsx

```diff
@@ -188,31 +183,10 @@ const StatefulEventComponent: React.FC<Props> = ({
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   const indexName = event._index!;
 
-  const onToggleShowNotes = useCallback(() => {
-    if (!expandableFlyoutDisabled && securitySolutionNotesEnabled) {
-      openFlyout({
-        right: {
-          id: DocumentDetailsRightPanelKey,
-          params: {
-            id: eventId,
-            indexName,
-            scopeId: timelineId,
-          },
-        },
-        left: {
-          id: DocumentDetailsLeftPanelKey,
-          path: {
-            tab: LeftPanelNotesTab,
-          },
-          params: {
-            id: eventId,
-            indexName,
-            scopeId: timelineId,
-          },
-        },
-      });
-    } else {
-      setShowNotes((prevShowNotes) => {
+  const onToggleShowNotesHandler = useCallback(
+    (currentEventId?: string) => {
+      onToggleShowNotes?.(currentEventId);
+      setFocusedNotes((prevShowNotes) => {
```

Comment:
Just curious, why use a state handler vs a ref here?

Reply:
So this whole logic was already there. I just renamed `setShowNotes` to `setFocusedNotes` as the previous name was misleading. Was 100% sure yet, what and why it was doing except focusing notes icon, so I left it as it is.

Reply:
Lol, gotta love that. Well cleaning up tech debt is gonna be fun in the future 😂